### ISSUE TITLE
Disqus also shows on the 404 page

### ISSUE THREAD

**Issue by JC5** [squidfunk/mkdocs-material#2186]
- [x] I've read the [contribution guidelines][1] and agree with them

__I've found a bug and checked that ...__

- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... the documentation does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem

## Description

The 404 page also renders a Disqus block, which I believe should be disabled on 404 pages. Or at least optional?

### Package versions

* Python 3.8.5
* mkdocs, version 1.1.2
* material Version: 6.2.3

``` yaml
theme:
  name: material
  features:
    - navigation.tabs
  palette:
    primary: blue grey
    accent: red
  logo: img/logo.png
  favicon: img/favicon.ico
  icon:
    repo: fontawesome/brands/github
extra:
  disqus: firefly-iii-documentation
  social:
    - icon: fontawesome/brands/twitter
      link: https://twitter.com/Firefly_III
    - icon: fontawesome/brands/github
      link: https://github.com/firefly-iii
    - icon: fontawesome/brands/reddit
      link: https://old.reddit.com/r/FireflyIII

plugins:
  - search
  - 'git-revision-date'
  - redirects:
      redirect_maps:

```

### System information

MacOS, Firefox 84.0.1

**Comment by squidfunk** [comment]
That's indeed unintended, we should exclude `disqus` from the `404.html`.

As a quick fix you can use [theme overrides](https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme) and override the `disqus` block:

``` jinja
<!-- overrides/404.html -->
{% extends "main.html" %}
{% block disqus %}{% endblock %}
{% block content %}
  <h1>404 - Not found</h1>
{% endblock %}
```

**Comment by squidfunk** [comment]
Fixed in 5b1ee2ac.

**Comment by squidfunk** [comment]
Released as part of 6.2.4.
